Sources/Classes/core/MiniApp.swift

```diff
@@ -206,8 +206,33 @@ public class MiniApp: NSObject {
         try? MiniAppSecureStorage.wipeSecureStorage(for: miniAppId)
     }
 
-    public static func unzipMiniApp(fileName: String, miniAppId: String, versionId: String) {
-        MiniAppSDKUtility.unzipMiniApp(fileName: fileName, miniAppId: miniAppId, versionId: versionId)
+    /// Unzip the provided archive file from bundle.
+    /// - Parameters:
+    ///   - fileName: File name of the Bundle that you want to extract in MiniApp folder
+    ///   - miniAppId: MiniApp ID
+    ///   - versionId: Version ID
+    ///   - completionHandler: Status of the Unzip feature
+    public static func unzipMiniApp(fileName: String, miniAppId: String, versionId: String, completionHandler: ((Result<Bool, MASDKError>) -> Void)? = nil) {
```

Comment:
```suggestion
    public static func unzipMiniApp(fileName: String, miniAppId: String, versionId: String, completionHandler: @escaping ((Result<Bool, MASDKError>) -> Void)? = nil) {
```

Reply:
@Climbatize , closure is already optional. So we don't need to add escaping